server/src/main/java/io/spine/server/entity/storage/Columns.java

```diff
@@ -147,94 +107,158 @@ public static void checkColumnDefinitions(EntityClass<?> entityClass) {
     static EntityColumn findColumn(Class<? extends Entity> entityClass, String columnName) {
         checkNotNull(entityClass);
         checkNotEmptyOrBlank(columnName, "entity column name");
-        ensureRegistered(entityClass);
 
-        final Collection<EntityColumn> cachedColumns = getColumns(entityClass);
-        for (EntityColumn column : cachedColumns) {
+        final Collection<EntityColumn> entityColumns = obtainColumns(entityClass);
+        for (EntityColumn column : entityColumns) {
             if (column.getName()
-                      .equals(columnName)) {
+                    .equals(columnName)) {
                 return column;
             }
         }
 
         throw new IllegalArgumentException(
-                format("Could not find a EntityColumn description for %s.%s.",
-                       entityClass.getCanonicalName(),
-                       columnName));
+                format("Could not find an EntityColumn description for %s.%s.",
+                        entityClass.getCanonicalName(),
+                        columnName));
     }
 
     /**
      * Retrieves {@linkplain EntityColumn columns} for the given {@code Entity} class.
      *
+     * <p>Performs checks for entity column definitions correctness along the way.
+     *
+     * <p>If check for correctness fails, throws {@link IllegalStateException}.
+     *
      * @param entityClass the class containing the {@link EntityColumn} definition
      * @return a {@link Collection} of {@link EntityColumn} corresponded to entity class
+     * @throws IllegalStateException if entity column definitions are incorrect
      */
-    static Collection<EntityColumn> getColumns(Class<? extends Entity> entityClass) {
+    static Collection<EntityColumn> obtainColumns(Class<? extends Entity> entityClass) {
```

Comment:
There is a lot of similar methods, which aim is to read/extract/obtain the columns. I would create an inner (?, or top-level?) class (such as `ColumnReader` or something) and use its instance (i.e. its instance methods!) instead of chaining the static method calls.

Also, there is another point. Some of these methods are package-private by design, others look `private`, but they are still package-private (most likely, for testing). I think this issue will fade away once the methods are extracted to one or more of such helper classes.